package.json

```diff
@@ -58,11 +58,11 @@
     "react-native-background-timer": "2.0.0",
     "react-native-calendar-events": "github:jitsi/react-native-calendar-events#dabfabc4bacc1424a8b93ebdda6f3820dee198cb",
     "react-native-callstats": "3.27.0",
-    "react-native-fetch-blob": "github:flatfox-ag/react-native-fetch-blob#exception_fixes",
+    "react-native-fetch-blob": "github:joltup/react-native-fetch-blob#0.10.9",
```

Comment:
Xcode "Version 9.4 (9F1027a)" tells me: /Users/lmarinov/Documents/atlassian-video-engineering/jitsi-meet-react/node_modules/react-native-fetch-blob/ios/RNFetchBlob/RNFetchBlob.m:77:1: Duplicate declaration of method 'requiresMainQueueSetup'

Indeed, there's:

````objective-c

+ (BOOL)requiresMainQueueSetup {
    return NO;
}
````

And later on there's:
````objective-c

+ (BOOL)requiresMainQueueSetup
{
    return YES;
}
````

I took a brief look and got the impression they fixed it later on.